Dockerfile

```diff
@@ -1,40 +1,51 @@
-FROM adoptopenjdk/openjdk11:jdk-11.0.7_10-slim
+FROM adoptopenjdk:11.0.8_10-jdk-hotspot as cruisecontrol
 ARG VERSION=2.5.13
+WORKDIR /
 USER root
-RUN apt-get update && apt-get install -y --no-install-recommends git ca-certificates
-RUN git clone -b ${VERSION} https://github.com/linkedin/cruise-control.git
-RUN cd cruise-control && ./gradlew jar copyDependantLibs
+RUN \
+  set -xe; \
+  apt-get update -qq \
+  && apt-get install -qq --no-install-recommends \
+    git ca-certificates
+RUN \
+  set -xe; \
+  git clone --branch ${VERSION} --single-branch https://github.com/linkedin/cruise-control.git \
```

Comment:
I'd use `--depth 1` instead of `--single-branch`

Reply:
@schultheiszn Agreed. Fixed.